freqtrade/freqtradebot.py

```diff
@@ -72,6 +73,28 @@
 logger = logging.getLogger(__name__)
 
 
+# Once validated move that in a dataclass folder?
+@dataclass
+class OrderToCreate:
+    pair: str
+    type: Literal['limit', 'market']
+    side: Literal['buy', 'sell']
+    price: float
+    amount: float
+    stake_amount: float
+    leverage: float
+    order_tag: str
+    reduce_only: bool
```

Comment:
This is implied - if it's an exit order, it's a reduce only.
the calculations won't support position reversals without closing the position.

leverage can also not actually change mid-trade. 
That's assumptions the calculations make.

Reply:
Yes, so it is not needed to keep reduce_only param in OrderToCreate class right?

Reply:
For now OrderToCreate store the parameter value, it can be implied or retreived from a callback, that's why reduce_only is present here